### DISCUSSION TITLE
QcefView调用的网页地址拖拽无效

### DISCUSSION THREAD

**Discussion by junhunlrw** [CefView/QCefView#204] [General]
用这个UI框架调用了一个网页，如果单纯在浏览器访问网页，里面的拖拽效果是可用的，但是放到这个框架引入之后，拖拽效果失效了，请问什么原因？大佬们，有懂的话求回复

**Comment by sdcarterchen** [comment]
OSR模式下这个需要自己实现

**Reply by L-Super** [reply]
> OSR模式下这个需要自己实现

应该怎样实现呢，大佬可否提供一个思路
